src/components/widgets/AccountInfo.tsx

```diff
@@ -1,12 +1,14 @@
 import { AccountRecord, CollectionPage, OperationRecord } from 'js-kinesis-sdk'
 import { startCase } from 'lodash'
 import * as React from 'react'
+import { Connection } from '../../types'
 import { renderAmount } from '../../utils'
 import { HorizontalLabelledField } from '../shared'
 import { OperationList } from './OperationList'
 
 interface Props {
-  account: AccountRecord,
+  account: AccountRecord
```

Comment:
missing a comma?
`account: AccountRecord,` ?

Reply:
interfaces don't need commas, but it's good to put them in - including the final line - so that it's consistent across the app and so that git doesn't mark changes like this that aren't real changes

Reply:
eg.
```suggestion
interface Props {
  account: AccountRecord,
  selectedConnection: Connection,
}
```